Classes/Controller/SearchController.php

```diff
@@ -203,6 +205,7 @@ public function makeFacetsMenuArray($facets)
         $menuArray = [];
         // Set default value for facet search.
         $search = [
+            'query' => '*:*',
```

Comment:
When using "OR" in the extended search with no search query, this would result in a Solr query `*:* OR title_tsi:...`. So the extended search would basically be ignored. (Is this why in 0e23cb8 you removed it in the first place?)

Reply:
You're right. This happens. The whole "extended search" is not well designed. E.g. it is unusable, if you make a full text search.
In our catalogue, the simple search is hidden, if you use the extended search. Maybe this should be done in Kitodo.Presentation, too. I reset any query in `makeFacetsMenuArray()` if using the extended search now. Hopefully this doesn't destroy anything else...